homeassistant/components/moehlenhoff_alpha2/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Möhlenhoff Alpha 2",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/moehlenhoff_alpha2",
-  "requirements": ["moehlenhoff-alpha2==1.2.1"],
+  "requirements": ["moehlenhoff-alpha2==1.3.0"],
```

Comment:
We recommend creating separate PRs for library bumps and only include other changes that are required for the version bump in the same PR.

Reply:
I created PR #85720 for the version bump.